queue_services/entity-filer/src/entity_filer/filing_processors/annual_report.py

```diff
@@ -24,7 +24,7 @@ def process(business: Business, filing: Dict):
     """Render the annual_report onto the business model objects."""
     agm_date = filing['annualReport'].get('annualGeneralMeetingDate')
     ar_date = filing['annualReport'].get('annualReportDate')
-    if agm_date and validations.annual_report.requires_agm(business):
+    if agm_date and validations.annual_report.requires_agm(business):  # pylint: disable=no-member;
```

Comment:
why is this lint disabled here?

Reply:
because pylint was throwing up an error for requires_agm not being a member (which it is)

Reply:
We should check how you're running lint, passes here. 
Our convention is to try and leave a comment as to why we're overriding / disabling something for future reviewers.
I think you can either remove this, or place a comment as to why it was added.